src/transactions/feed/queryHelper.ts

```diff
@@ -210,16 +263,67 @@ export function useFetchTransactions(): QueryHookResult {
   }
 }
 
-async function queryTransactionsFeed(
-  address: string | null,
-  localCurrencyCode: string,
+function anyChainHasMorePages(pageInfo: { [key in Chain]?: PageInfo | null }): boolean {
+  return Object.values(pageInfo).reduce((acc, val) => acc || !!val?.hasNextPage, false)
```

Comment:
```suggestion
  return Object.values(pageInfo).some(({ hasNextPage }) => !!hasNextPage)
```
code golf

Reply:
not sure if it compiles, but the main point is that "some" could be used here instead of reduce